spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReaderDeletes.java

```diff
@@ -80,9 +88,9 @@ public TestSparkReaderDeletes(boolean vectorized) {
 
   @Parameterized.Parameters(name = "vectorized = {0}")
   public static Object[][] parameters() {
-    return new Object[][] {
-        new Object[] {false},
-        new Object[] {true}
+    return new Object[][]{
+        new Object[]{false},
+        new Object[]{true}
```

Comment:
can revert it back if it's un-intentional

Reply:
Ah I see, my IDE (using Intellij) here doesn't reformat this though, I am using this [iceberg-java-style](https://iceberg.apache.org/contribute/#setting-up-ide-and-code-style) . either ways, I think it should be fine if it's aligned to [iceberg-java-style](https://iceberg.apache.org/contribute/#setting-up-ide-and-code-style) :) !!!

Also just realized, we can re-write this to as well : 
```java
    return new Object[] {
        new Object[] {false, true};
    }
```

Reply:
Right, we don't actually need a 2D array here.